packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -483,12 +479,9 @@ export const Slider = React.forwardRef((props, ref) => {
   return (
     <Box
       ref={ref}
-      className={cx(
-        'iui-slider-component-container',
-        `iui-slider-${orientation}`,
-        { 'iui-disabled': disabled },
-        className,
-      )}
+      className={cx('iui-slider-container', className)}
+      data-iui-orientation={orientation}
+      data-iui-disabled={disabled}
```

Comment:
generally you want to use `undefined` for the falsy case, to avoid adding unnecessary data attribute

```suggestion
      data-iui-disabled={disabled ? 'true' : undefined}
```